api/src/main/java/org/openmrs/module/facilitydata/model/CodedFacilityDataQuestionType.java

```diff
@@ -13,17 +13,24 @@
  */
 package org.openmrs.module.facilitydata.model;
 
+
+import javax.persistence.*;

```

Comment:
We do not encourage wild card imports https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports